web/README.md

```diff
@@ -0,0 +1,295 @@
+# @thorvg/lottie-player
+
+A Lottie Player which uses [ThorVG](https://github.com/thorvg/thorvg) as a renderer, provides web component for easily embedding and playing [Lottie](https://airbnb.io/lottie) animations.
+
+# Installation
+
+- Import from CDN
+```html
+<script src="https://unpkg.com/@thorvg/lottie-player@latest/dist/lottie-player.js"></script>
+```
+
+- Install from NPM
+```sh
+npm install @thorvg/lottie-player
+```
+
+# Usage
+### With HTML (Basic Usage)
+
+Once you import from CDN, you can access `<lottie-player/>`
+
+```html
+<lottie-player 
+  autoPlay 
+  loop
+  mode="normal"
+  src="https://lottie.host/6d7dd6e2-ab92-4e98-826a-2f8430768886/NGnHQ6brWA.json"
```

Comment:
@hermet I used any file, we should have valid. Did you have any free-license lottie file? so we can use on docs or example, can I have one?

Reply:
@tinyjin Let's use this: https://lottiefiles.com/animations/property-market-JJWInHbEZn
As it is distributed under the [Lottie Simple License](https://lottiefiles.com/license), there is no requirement to credit the author or provide a source link. However, it might be better to reference the above [Lottie Simple License](https://lottiefiles.com/license). Optionally, you can also mention the author's name, Mark Airs but it's not mandatory.